service/operator/operator.go

```diff
@@ -1,127 +0,0 @@
-package operator
```

Comment:
All that got removed because the implementation moved to the operatorkit framework to get rid of some bootstrap magic in the operator projects.